internal/v1/handler_blueprints.go

```diff
@@ -27,6 +30,7 @@ var (
 	blueprintNameRegex         = regexp.MustCompile(`\S+`)
 	customizationUserNameRegex = regexp.MustCompile(`\S+`)
 	blueprintInvalidNameDetail = "The blueprint name must contain at least two characters."
+	now                        = time.Now
 )
```

Comment:
Oh here is a bug - this time is now a fixed time. In fact it is the time when pod was started and it will never be updated. Package-level variables are initialized during initialization of the process.

I see what you are trying to do, can't you just keep the code unchanged but in the test itself before you store or compare the JSON simply modify it and set the time to a fixture? Like a birthday of your faviourite movie star or something :-)

You can parse arbitrary JSON into an empty interface map in Go pretty easily, then you can manipulate it too. I think it is easier than creating a full stub for this handler.

Reply:
Thanks for catching this